brain/experiment.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2013-2018, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+#pragma once
+
+#include <brain/api.h>
+#include <brain/types.h>
+
+namespace brain
+{
+class Experiment
+{
+public:
+    /**
+     * Open a experiment configuration
+     *
+     * @param source the URI to the BlueConfig file.
+     */
+    BRAIN_API explicit Experiment(const URI& source);
```

Comment:
This is 'enabling people' again. There are already new workflows which provide no BlueConfig anymore, hence this is adding a deprecated API. I would vote if you need that in RTNeuron, put it there. From the tools we have today, this is not needed elsewhere atm.

Reply:
I'll remove it from this code review, but concept of the experiment is valid and will reappear with simulation configs from Sonata. The mistake that was made in the past was using a single file for both circuit and simulation config. This has been addressed with SONATA and we will have to support it one way or another.